__tests__/app.test.js

```diff
@@ -83,6 +85,7 @@ describe("/api/articles/:article_id", () => {
   it("Reponds with 'bad request' as error message if article id does not exist", () => {
     return request(app)
       .get("/api/articles/306")
+      .expect(404)
```

Comment:
To be really picky on your test description, is it a bad request? If it doesn't exist but is a valid number that is a valid request.